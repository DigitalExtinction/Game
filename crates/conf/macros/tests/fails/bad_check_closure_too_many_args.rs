use conf_macros::Config;

#[derive(Config)]
pub struct TestConfig {
    pub field1: u32,
    #[check(|foo: u32, bar:u32| todo!())]
    pub field2: u32,
    pub field3: u32,
}

fn main() {}
