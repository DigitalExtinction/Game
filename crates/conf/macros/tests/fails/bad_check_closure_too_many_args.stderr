error: expected a closure with one argument
 --> tests/fails/bad_check_closure_too_many_args.rs:6:14
  |
6 |     #[check(|foo: u32, bar:u32| todo!())]
  |              ^^^^^^^^^^^^^^^^^
